arangod/Pregel/Actor/include/Actor/ActorPID.h

```diff
@@ -69,11 +69,14 @@ using DatabaseName = std::string;
 
 struct ActorPID {
   ServerID server;
+  DatabaseName database;
```

Comment:
here is some architectural change in this PR that is not related to using Actors? 

In particular actors are now tied to databases?

Reply:
I think this is needed to use actors in ArangoDB: REST messages can only be send to specific databases, so the database name really needs to be part of the actor address, otherwise we cannot send messages to other servers. I tried to do it differently but failed, if you have a better idea I'm up for it.